README.md

```diff
@@ -2,105 +2,26 @@
 ![](SlicerCART/src/Resources/Icons/SlicerCART.png)
 SlicerCART: Configurable Annotation and Review Tool. 
 
-## Description
+This is a 3D Slicer extension aimed to improve manual 
+segmentation, classification and revision workflows.
```

Comment:
Maybe add a decsription of what the module does exactly, if someone arrives on the README page, hard to say what is the purpose of the module

Reply:
Ok description sentence added. Better?

Reply:
where is it ? could you link with a commit ?

Reply:
yes. I addressed your comments and now will commit. Thanks!

Reply:
Ok! Got it. So I understand, each comment addressed = 1 commit with link of the commit is that right? For that above,should be addressed in https://github.com/neuropoly/slicer-manual-annotation/pull/105/commits/bc3120a4fec1f65a626ec5348e6e165306ea87f8